### DISCUSSION TITLE
Manual Pricing and % of medicare for outpatient claims

### DISCUSSION THREAD

**Discussion by jwerner15083** [CMSgov/price-transparency-guide#749] [General]
I have not visited our MRF files in a while, but we found an issue that we need to solve.  Many of our providers are "manually" priced, meaning that their rate structure is unable to be cleanly represented in a MRF.   OR the schedule is based on a percent of medicare for Outpatient hospital, which I currently do not know how to represent on a MRF (see below).   Since these providers do not have rate structures being represented on the MRF, the providers are not showing on the provider lookups that external vendors are using.  Strangely we provide a provider lookup tool, but our clients are deferring to the Zelis/Talon/and other companies who are ingesting the MRFs.

My question(s):
   -  Is there a standard way to show manually priced claims as per the MRF. If so can someone give a quick simple example?  I know we did this for the percent of charge claims...can we also get an example for Manual pricing?  
   - Second...has anyone had experience showing rates based on the APC or % of Medicare for OP Hospital?  With IP I can enumerate all the DRGs, but with OP I simply do not have the access to the MRF data that I would have expected/hoped.  So I am looking for some outside information to determine the data structure.  For example if I have a OP hospital accepting 150% of Medicare is there a simpler way to make this display?  Or am I stuck trying to sort out all the APCs and the CMS value for that APC (for that provider), etc.  I am not sure I have that info!

Thanks!  Jason

**Comment by tdfow** [comment]
For the manual pricing that is based on a percentage of Medicare, then could you not publish the rates as negotiated_type of percentage and provide the percentage in the negotiated_rate, then use the additional_information field to state it would be based on noted percentage of the Medicare rate?
If the manual price is based on a combination of items, I would think that seems to align with a 'derived' negotiated_type, but there would have to be a routine to identify the 'derived' pricing to associate with the service/provider data.  The additional_information field could refer to the manual related pricing or the needed context to further clarify the pricing information.

**Reply by jwerner15083** [reply]
Thanks!  I think I will do that.

**Comment by shaselton-usds** [comment]
Hello, just following up on this. If you have a negotiated rate that is a percentage that is based on Medicare, you need to do the math to get to an actual dollar amount. The Medicare fee schedule is public and it shouldn't be too difficult to convert a percentage of Medicare arrangement to a dollar amount.

**Reply by jwerner15083** [reply]
@shaselton-usds  For the professional (lab, professional, drug, dme) that is fine and fairly easy to do.  Just need the values to plug in.

My question was about the outpatient/APC values.  While the APC values are public, the rules surrounding payment are so complex that even CMS has not been able to create a OPPS web/pc pricer.   The bundling rules, variations in provider amounts, comprehensive APC values,  etc all "conspire" against accurate coding and reimbursement rates.  There are codes payable when billed alone, but are not payable when billed with others, etc.  

I can and do provide the simple HCPCS to APC to payment % amount.  But we have companies like Talon who are complaining that it is not accurate enough and not providing the correct reimbursement.  I have to agree because OPPS is a formula not a code by code value.  Has CMS addressed this in any official way?
